src/components/Loader/Loader.module.scss

```
@@ -0,0 +1,21 @@
+@import "@config/variables";
+
+.loader {
+  border: 2px solid $main-color;
+  border-radius: 50%;
+}
+
+.loader_size-s {
+  width: 20px;
+  height: 20px;
+}
+
+.loader_size-m {
+  width: 40px;
+  height: 40px;
+}
+
+.loader_size-l {
+  width: 60px;
+  height: 60px;
+}
```

так как подключен `scss`, то можно
```suggestion
.loader {
  border: 2px solid $main-color;
  border-radius: 50%;
  
  &_size-l {
  width: 60px;
  height: 60px;
}
}
```

и тд...